src/lib/caching/stores/GuildEmojiStore.ts

```diff
@@ -109,7 +114,7 @@ export interface GuildEmojiStoreAddData {
 	/**
 	 * The 128x128 emoji image.
```

Comment:
Are you sure they're just 128x128?

Reply:
From the [docs](https://discord.com/developers/docs/resources/emoji#create-guild-emoji-json-params), sir.